components/dashboard/src/App.tsx

```diff
@@ -90,6 +90,13 @@ function App() {
         }
     }, []);
 
+    useEffect(() => {
+        (async () => {
+            const teams = await getGitpodService().server.getTeams();
+            console.log('Teams:', teams);
```

Comment:
Note: This throws when not logged in -- a try/catch could be cleaner.

Reply:
Can we leave this test code out of the commit?